refrigerator.cs

```diff
@@ -136,29 +129,29 @@ public List<Item> getYourFoods(Kashrot kashrot, TypeItem typeItem)
             {
                 yourItems.AddRange(shelf.getYourFood(kashrot, typeItem));
             }
-            ;
 
             return yourItems;
         }
-        
+
         public List<Shelf> sortShelfesInFrige()
         {
             List<Shelf> items = new List<Shelf>();
```

Comment:
also here, remove the definition of the list to item obj and use it as property.